tests/test_core.py

```diff
@@ -282,6 +287,44 @@ def test_fetch_attachment_content(self):
                 )
             )
 
+    def test_add_attachment_content(self):
+        """Tests add_attachment_content method success."""
+        # slims_api is dynamically added to slims client
+        mock_response = MagicMock()
+        mock_response.text.return_value = self.example_add_attachments_response_text
+        with patch.object(
+            self.example_client.db.slims_api,
+            "post",
+            return_value=mock_response,
+        ) as mock_post:
+            self.example_client.add_attachment_content(
+                SlimsUnit(
+                    unit_name="test",
+                    unit_pk=1,
+                ),
+                "test",
+                "some test content",
+            )
+            assert mock_post.call_count == 1
```

Comment:
Similar thing here where the other tests use `self.assertEqual(1, mock_post.call_count)`. It might be helpful to add an additional check on the exact call by seeing what `mock_post.mock_calls` is doing and then adding `mock_post.assert_has_calls()`

Reply:
Addressed here: [ea62847](https://github.com/AllenNeuralDynamics/aind-slims-api/pull/21/commits/ea628472bfbdd08b24804ca52a59115d61481c43)